docs/source/flow-project.rst

```diff
@@ -215,7 +268,7 @@ As shown before, all *eligible* operations can then be executed with:
 
     ~/my_project $ python project.py run
 
-The status determination operates in serial by default, because typically the overhead costs of using threads/processes are large. However this can be configured by setting a value for the ``flow.status_parallelization`` configuration key.
+The status determination operates in series by default, because typically the overhead costs of using threads/processes are large. However this can be configured by setting a value for the ``flow.status_parallelization`` configuration key.
```

Comment:
I'm not sure if this change is correct. To the best of my knowledge, "serial execution" is a term, but "series execution" is not a term.

Reply:
Bradley and I discussed offline. We agreed that series is the noun form of serial but that the adjective form seems to be the norm in the language of algorithms. Referencing Wikipedia and the OED (serial and sequential 2b), we agreed to switch to "Status is determined sequentially..."

Wikipedia:
>Concurrency and parallelism are in general distinct concepts, but they often overlap – many distributed algorithms are both concurrent and parallel – and thus "sequential" is used to contrast with both, without distinguishing which one. If these need to be distinguished, the opposing pairs sequential/concurrent and serial/parallel may be used.

OED:
> seqential def 2b: Of, pertaining to, or designating various aspects of a computer system and its control programming that operate or are utilized serially; "

Reply:
```suggestion
Status is determined sequentially by default, because typically the overhead costs of using threads/processes are large. However, this can be configured by setting a value for the ``flow.status_parallelization`` configuration key.
```